### ISSUE TITLE
False "Non-default argument follows default argument" error

### ISSUE THREAD

**Issue by ods** [microsoft/pyright#733]
The following code is valid Python:
```python
def f(*a, b=1, c):
    pass
```
But pyright 1.1.42 produces an error for it:
```
1:16 - error: Non-default argument follows default argument
```

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next published version of Pyright.

**Comment by erictraut** [comment]
This is fixed in Pyright 1.1.43, which I just published.
